src/index.html

```diff
@@ -17,6 +17,90 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__container">
+        <div class="header__logo">
+          <a
+            href="#moyo"
+            class="header__logo-link"
+          >
```

Comment:
The attributes of the `a` element should be formatted on new lines with 2-space indentation related to the tag. The closing bracket should be aligned with the opening tag.